src/DataViews/include/DataViews/LiveFunctionsDataView.h

```diff
@@ -58,11 +59,12 @@ class LiveFunctionsDataView : public DataView {
                                              const std::vector<int>& selected_indices) override;
   void DoFilter() override;
   void DoSort() override;
-  [[nodiscard]] uint64_t GetInstrumentedFunctionId(uint32_t row) const;
+  [[nodiscard]] uint64_t GetScopeId(uint32_t row) const;
   [[nodiscard]] std::optional<orbit_client_data::FunctionInfo>
   CreateFunctionInfoFromInstrumentedFunction(
       const orbit_grpc_protos::InstrumentedFunction& instrumented_function);
 
+  // Maps scope_ids corresponding to dynamically instrumented functions to FunctionInfo instances
   absl::flat_hash_map<uint64_t, orbit_client_data::FunctionInfo> functions_{};
```

Comment:
I think that renaming to `scope_id_to_function_` or `functions_by_scope_id_` can help more than the comment.

Reply:
I'd still keep the comment. It points out, only the scope_ids corresponding to functions are present.

Reply:
Definitely, even better.